crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -637,7 +633,7 @@ async fn get_filtered_config(
         })?;
     let mut query_params_map: serde_json::Map<String, Value> = Map::new();
```

Comment:
@ayushjain17 Can we remove this handler? I don't think this is being used anywhere, getConfig can be used for the same purpose.

Reply:
Sure, redirecting both the endpoints to the same url
or should I deprecate the filter endpoint ?

Reply:
I think we should deprecate this endpoint as it has the same logic as the above.